public/interactive-examples/wm2-count-revenue/counter.js

```diff
@@ -2,19 +2,19 @@ let total = 0;
 
 if (window.MonetizationEvent) {
   const link = document.querySelector('link[rel="monetization"]');
+  const contribution = document.getElementById('contribution');
 
-  link.addEventListener("monetization", (event) => {
+  link.addEventListener('monetization', (event) => {
     const {
       amountSent: { value, currency },
     } = event;
     console.log(`Browser sent ${currency} ${value}.`);
 
-    if (total === 0) {
-      document.getElementById("currency").innerText = currency;
-    }
-
     total += Number(value);
 
-    document.getElementById("total").innerText = total.toFixed(9);
+    contribution.innerText = new Intl.NumberFormat('en-US', {
+      style: 'currency',
+      currency,
+    }).format(total);
```

Comment:
While it's true the spec uses 3-letter ISO country codes, some niche Web Monetization wallets are using custom 5-letter codes (for non-standard virtual currencies). So, using `Intl.NumberFormat` will crash there (it gracefully handles any 3-letter code, even if not a standard one).
Not sure if it's a good idea to let this complication be handled here (to prevent uncaught exception in event callback), but leaving as a note for future.